Utilities/XrdAdaptor/src/XrdRequestManager.cc

```diff
@@ -118,6 +121,50 @@ static void SendMonitoringInfo(XrdCl::File &file) {
   }
 }
 
+std::string *GetQueryTransport(const XrdCl::URL &url, uint16_t query) {
+  XrdCl::AnyObject result;
+  XrdCl::DefaultEnv::GetPostMaster()->QueryTransport(url, query, result);
+  std::string *method;
+  result.Get(method);
+  return method;
+}
+
+static void TracerouteRedirections(const XrdCl::HostList *hostList) {
```

Comment:
I'd prefer anonymous namespace over file-`static`. Please start the function name with a lower-case letter.